<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Collage Maker</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="PaintArea" name="paintArea" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar_dibujo"/>
   </widget>
   <widget class="QMenu" name="menuOpciones_de_collage">
    <property name="title">
     <string>Opciones de collage</string>
    </property>
    <addaction name="actionChange_resolution"/>
    <addaction name="actionChange_Subdivitions"/>
    <addaction name="actionChange_random_pick"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_images"/>
    <addaction name="actionGenerate_collage"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuOpciones_de_collage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Abrir una imagen</string>
   </property>
  </action>
  <action name="actionGuardar_dibujo">
   <property name="text">
    <string>Guardar dibujo</string>
   </property>
  </action>
  <action name="actionChange_resolution">
   <property name="text">
    <string>Cambiar resolucion</string>
   </property>
  </action>
  <action name="actionChange_Subdivitions">
   <property name="text">
    <string>Cambiar subdivisiones</string>
   </property>
  </action>
  <action name="actionChange_random_pick">
   <property name="text">
    <string>Cambiar random pick</string>
   </property>
  </action>
  <action name="actionSelect_images">
   <property name="text">
    <string>Seleccionar imagenes</string>
   </property>
  </action>
  <action name="actionGenerate_collage">
   <property name="text">
    <string>Generar collage</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QWidget</extends>
   <header location="global">PaintArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
